perfkitbenchmarker/linux_packages/hpcc.py

```diff
@@ -50,6 +77,16 @@ def _Install(vm):
       (re.escape(openblas.OPENBLAS_DIR), HPCC_MAKEFILE_PATH))
   vm.RemoteCommand(sed_cmd)
   vm.RemoteCommand('cd %s; make arch=Linux_PII_CBLAS' % HPCC_DIR)
+  vm.RemoteCommand('cp %s/hpcc hpcc' % HPCC_DIR)
+
+
+def _Install(vm):
+  """Installs the HPCC package on the VM."""
+  vm.Install('openmpi')
+  if __name__.split('.')[-1] in FLAGS.override_packages:
+    _Override(vm)
+  else:
+    _Build(vm)
```

Comment:
I bet this logic could be moved to `LinuxVirtualMachine.Install` so that it could be shared among override-able packages.